src/funcCoreTools/validateFuncCoreToolsInstalled.ts

```diff
@@ -61,7 +85,8 @@ export async function validateFuncCoreToolsInstalled(context: IActionContext, me
 
     // validate that Func Tools was installed only if user confirmed
     if (input === install && !installed) {
-        if (await context.ui.showWarningMessage(localize('failedInstallFuncTools', 'The Azure Functions Core Tools installation has failed and will have to be installed manually.'), DialogResponses.learnMore) === DialogResponses.learnMore) {
+        ext.outputChannel.show();  // Make it easier for the user to find the last command in case they want to re-run with 'sudo'
+        if (await context.ui.showWarningMessage(failedInstall, DialogResponses.learnMore) === DialogResponses.learnMore) {
```

Comment:
Adding the comment here too since the other thread was resolved;

Can we add a button to copy the command to the clipboard?

Reply:
Updated: 

![image](https://user-images.githubusercontent.com/40250218/205410666-91d78275-86dc-44d3-9d3f-6a7c86db86dc.png)